core/frontend/src/tools/AccuDrawTool.ts

```diff
@@ -946,59 +946,60 @@ export class AccuDrawShortcuts {
   }
 
   /** @internal Temporary keyboard shortcuts. */
-  public static processShortcutKey(keyEvent: KeyboardEvent): boolean {
-    switch (keyEvent.key.toLowerCase()) {
-      case "enter":
-        AccuDrawShortcuts.lockSmart();
-        return true;
-      case "x":
-        AccuDrawShortcuts.lockX();
-        return true;
-      case "y":
-        AccuDrawShortcuts.lockY();
-        return true;
-      case "z":
-        AccuDrawShortcuts.lockZ();
-        return true;
-      case "a":
-        AccuDrawShortcuts.lockAngle();
-        return true;
-      case "d":
-        AccuDrawShortcuts.lockDistance();
-        return true;
-      case "m":
-        AccuDrawShortcuts.changeCompassMode();
-        return true;
-      case "t":
-        AccuDrawShortcuts.setStandardRotation(RotationMode.Top);
-        return true;
-      case "f":
-        AccuDrawShortcuts.setStandardRotation(RotationMode.Front);
-        return true;
-      case "s":
-        AccuDrawShortcuts.setStandardRotation(RotationMode.Side);
-        return true;
-      case "v":
-        AccuDrawShortcuts.setStandardRotation(RotationMode.View);
-        return true;
-      case "o":
-        AccuDrawShortcuts.setOrigin();
-        return true;
-      case "c":
-        AccuDrawShortcuts.rotateCycle();
-        return true;
-      case "q":
-        AccuDrawShortcuts.rotateAxes(true);
-        return true;
-      case "e":
-        AccuDrawShortcuts.rotateToElement();
-        return true;
-      case "r":
-        AccuDrawShortcuts.defineACSByPoints();
-        return true;
-      default:
-        return false;
-    }
+  public static processShortcutKey(_keyEvent: KeyboardEvent): boolean {
+    // switch (keyEvent.key.toLowerCase()) {
```

Comment:
Are these all intentionally commented out?

Reply:
Yes. The AccuDraw shortcuts are being moved to the Keyboard Shortcuts system. It's a work in progress.

Reply:
Might be better to leave these for testing with display-test-app since I assume real shortcuts are part of ui-framework?

Reply:
Should this be a draft PR then so we're not annoying you with comments on unfinished code?

Reply:
Not sure how leaving them in will work because if I do that, both these AccuDraw shortcuts & the Keyboard shortcuts are activated.

Reply:
I'll abandon this PR. Obviously, some decisions need to be made about where the shortcuts are defined and activated.

Reply:
What I think needs to happen is that all current DgnPrimitiveTools that call processShortcuts should stop doing so (measuretools.ts, etc.). An override of ToolAdmin.onKeyTransistion in display-test-app can start calling it. Should probably rename processShortcuts in case something like the measure tools package that isn't in the monorepo is currently calling it so the call can be removed when the update versions.